plugin/src/main/resources/data/abilities.yml

```
@@ -29,4 +29,37 @@ abilities:
+    usage: right_click
+    hotbarItem: iron_shovel
+    displayItem: milk_bucket
+    metadata:
+      durationTicks: 160
+      propelTicks: 80
+      helixRadius: 1.5
+      centerSpeed: 0.9
+      playerSpeed: 1.0
+      damage: 5.0
+      maxTargets: 2
+      hitCooldownTicks: 10
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)